app/containers/Organisations/Organisations.tsx

```diff
@@ -148,10 +148,10 @@ export const OrganisationsComponent: React.FunctionComponent<Props> = (
               </>
             ) : (
               <Dropdown className="search-dropdown">
-                <Card.Title>
+                <h2>
                   Request access to apply for the CIIP on behalf of a Reporting
                   Operation:
-                </Card.Title>
+                </h2>
```

Comment:
Unfortunately headings accessibility won't always let us blindly use helpers like `<Card.Title>`, which arbitrarily sets it to a particular heading number (which may not be the number you need).